.github/workflows/check-untracked-repos.yml

```
@@ -0,0 +1,42 @@
+name: Check Untracked Repositories
+
+on:
+  schedule:
+    - cron: '0 */6 * * *'
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  check-untracked-repos:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-24.04
```
let's fix the runner version to avoid surprises